src/node.ts

```diff
@@ -0,0 +1,65 @@
+// This file attaches all functionality to Node
+// that would otherwise require circular dependencies.
+import Node from './node-internal'
+import Collection from './collection'
+import Literal from './literal'
+import { ValueType } from './types'
+
+export default Node
+
+/**
+ * Creates an RDF Node from a native javascript value.
+ * RDF Nodes are returned unchanged, undefined returned as itself.
+ * @method fromValue
+ * @param value - Any native Javascript value
+ */
+Node.fromValue = function (value: ValueType): Node | Literal | undefined | null | Collection {
+  if (typeof value === 'undefined' || value === null) {
+    return value
+  }
+  const isNode = Object.prototype.hasOwnProperty.call(value, 'termType')
+  if (isNode) {  // a Node subclass or a Collection
+    // @ts-ignore
+    return value
+  }
+  if (Array.isArray(value)) {
+    return new Collection(value)
+  }
+  return Literal.fromValue(value)
+}
+
+import Namespace from './namespace'
```

Comment:
I think usually it's good practice to put all `import` statements at the top of the file, so that it's clear when there are e.g. duplicate entries?